src/Obj.hs

```diff
@@ -558,6 +560,7 @@ instance Show Project where
             , "Search paths for 'load' command:\n    " ++ joinWith  "\n    " searchPaths
             , "Print AST (with 'info' command): " ++ if printTypedAST then "true" else "false"
             , "File path print length (when using --check): " ++ show filePathPrintLength
+            , "Don't compile:" ++ if noCompile then "true" else "false"
```

Comment:
Nit: Could we add a space at the end of the string?

Reply:
Sure thing.